addons/binding/org.openhab.binding.gpstracker/src/main/java/org/openhab/binding/gpstracker/internal/handler/TrackerHandler.java

```diff
@@ -105,10 +125,44 @@ public String getTrackerId() {
 
     @Override
     public void initialize() {
+        if (sysLocation != null) {
+            createBasicDistanceChannel();
+        } else {
+            logger.debug("System location is not set. Skipping system distance channel setup.");
+        }
+
         mapDistanceChannels();
         updateStatus(ThingStatus.ONLINE);
     }
 
+    /**
+     * Create distance channel for measuring the distance between the tracker and the szstem.
+     */
+    private void createBasicDistanceChannel() {
+        Iterator it = thing.getChannels().stream().filter(c -> CHANNEL_TYPE_DISTANCE.equals(c.getChannelTypeUID())).iterator();
+        @Nullable ThingHandlerCallback callback = getCallback();
+        if (!it.hasNext() && callback != null) {
+            Configuration config = new Configuration();
+            config.put(ConfigHelper.CONFIG_REGION_NAME, "System");
+            config.put(ConfigHelper.CONFIG_REGION_CENTER_LOCATION, sysLocation.toFullString());
+            config.put(ConfigHelper.CONFIG_REGION_RADIUS, 100);
+
+            ChannelUID homeDistanceChannelUID = new ChannelUID(thing.getUID(), "distance_system");
```

Comment:
You use camel case for all other channel IDs, so better do the same here: `distanceSystem`.